web_src/less/_repository.less

```diff
@@ -2821,6 +2821,73 @@ tbody.commit-list {
     padding-bottom: 8px;
 }
 
+.board {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    overflow-x: auto;
+    margin: 0 .5em;
+}
+
+.board-column {
+    background-color: rgba(0, 0, 0, .05) !important;
+    border: 1px solid rgba(34, 36, 38, .15) !important;
+    margin: 0 .5em !important;
+    padding: 5px !important;
+    width: 320px;
+    height: 60vh;
+    overflow-y: scroll;
+    flex: 0 0 auto;
+    overflow: visible;
+}
+
+.board-column > .cards {
+    margin: 0 !important;
+
```

Comment:
Need to slap a min-height on here of some kind, otherwise there's an issue in at least my browser with bootstrapping a new board because the droppable region is 0-height. My local version uses min-height: 35px so that it's not necessarily larger than the cards' default height, but it might be better to expand it further and de-couple card height from this.

Reply:
can you send a pull?

Reply:
Filed a pull request against the kanban_board branch here: https://github.com/adelowo/gitea/pull/9

Reply:
thx